website/docs/r/lb_icmp_monitor.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: "nsxt"
+page_title: "NSXT: nsxt_lb_icmp_monitor"
+sidebar_current: "docs-nsxt-resource-lb-icmp-monitor"
```

Comment:
this should be docs-nsxt-lb-monitor-resource-... since it is on a different sidebar I think